### PR TITLE
Feat/alert and button

### PR DESCRIPTION
**Pull request by tonymike99**
I have added the alert and button components.

### CODE REVIEW

File: Components/Alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+                <div class="alert danger">A simple danger alert!</div>
+
+                <div class="alert warning">A simple warning alert!</div>
+
+                <div class="alert info">A simple info alert!</div>
+
+                <div class="alert light">A simple light alert!</div>
+
+                <div class="alert dark">A simple dark alert!</div>
```
Comment (zoef-yt) [inline Components/Alert/alert.html:37]: where is the CSS code for all the classes for example primary, secondary, success, etc?

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: Components/Alert/alert.html (hunk 1 above)
Comment (tonymike99) [reply Components/Alert/alert.html:37]: The CSS for common classes are there in the components.css file, Zoef.

**Review by sanu29 — APPROVED** [review]
Clean and understandable code.  name contention used is continuous and meaningful.
No changes according to me.

**Comment by srishtigautam21** [comment]
everything looks good to me.....good work tonymike

File: Components/Alert/alert.html (hunk 1 above)
Comment (zoef-yt) [reply Components/Alert/alert.html:37]: ok.

**Review by zoef-yt — APPROVED** [review]

**Pull request merged**